arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -419,15 +394,6 @@ fun <A> nonEmptyListOf(head: A, vararg t: A): NonEmptyList<A> =
 inline fun <A> A.nel(): NonEmptyList<A> =
   nonEmptyListOf(this)
 
-@Deprecated(
-  "Kind is deprecated, and will be removed in 0.13.0. Please the plus method defined for NonEmptyList instead",
-  ReplaceWith(
-    "fix().plus(y.fix())",
-    "arrow.core.fix",
-    "arrow.core.plus"
-  ),
-  DeprecationLevel.WARNING
-)
 fun <A> NonEmptyList<A>.combineK(y: NonEmptyList<A>): NonEmptyList<A> =
```

Comment:
do we need this function at all anyway? 🤔

Reply:
Nope, we don't!